packages/devtools_app/lib/src/config_specific/flutter/import_export/import_export.dart

```diff
@@ -49,6 +56,52 @@ class ImportController {
         return;
     }
   }
+
+  void _navigateToImportScreen(DevToolsScreenType screenType) {
```

Comment:
This looks like it should be a method on the DevToolsScreenState.

Reply:
done